packages/react/src/analytics/integrations/Vitorino/Vitorino.ts

```diff
@@ -218,54 +224,26 @@ export default class Vitorino extends integrations.Integration {
       clientId,
       origin,
       customerId: getCustomerIdFromUser(data.user),
```

Comment:
Note: I am changing the type of `getCustomerIdFromUser` to return a `string` instead of a `number` because Omnitracking expects that customerId be a string instead of a number. If Vitorino needs that `customerId` be a number, then I would change this call to `parseInt(getCustomerIdFromUser(data.user) as string)` to guarantee that it will be a number (The `as string` cast will not be required when the task I am working on is merged, we will need to coordinate that).

Reply:
I've check the Vitorino docs and actually it accepts a string (for the guest `g_{id}`, so it's not a problem at all 🙂  updated, thanks!